components/database/mongodb-44/Makefile

```diff
@@ -0,0 +1,168 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Alexander Pyhalov
+# Copyright 2018 Michal Nowak
+# Copyright 2021 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+BUILD_BITS= 64
+BUILD_STYLE= justmake
+
```

Comment:
Does this also work with OpenSSL-1.1 if we set
USE_OPENSSL11= yes
here?

Reply:
I will check it out.

Reply:
Compiling with openssl 1.1 does not work

I extracted the failing test from scons.
Please see here: https://gist.github.com/grueni/7c3624e5b818a2ff6821a287deebe3f3

Reply:
Any idea?

Reply:
The missing symbol _Noreturn is defined in g++ starting with 11.